concepts/while-loops/about.md

```diff
@@ -0,0 +1,83 @@
+# About
+
+## General Syntax
+
+With a while loop you can execute code repeatably as long as a certain condition is fulfilled.
+
+It is written with the `while` keyword followed by a condition wrapped in round brackets and a code block that contains the _body_ of the loop wrapped in curly brackets.
+
+```javascript
+while (condition) {
+  // code that is executed repeatedly as long as the condition is true
+}
+```
+
+JavaScript also has a do-while loop.
+Here the condition is evaluated after the loop body was executed.
+This is useful when the condition depends on the outcome of the code in the body.
+
+```javascript
+do {
+  // the code here will always be executed once and then repeatedly
+  // if the condition is true after the last iteration
+} while (condition);
+```
+
+## Break
+
+Inside a loop body you can use the `break` keyword to stop the execution of the loop entirely.
+This is often used in combination with `true` as condition.
+With that, you can control when the loop should stop from any place inside the loop body.
+
+```javascript
+const winningNumber = 7;
+
+while (true) {
+  const num = readUserGuess();
+  if (num === winningNumber) break;
```

Comment:
```suggestion
  if (num === winningNumber) {
    break;
  }
```

Or did prettier do this :O :O :O ?

Reply:
no, no ... it was just my lazy developer self :see_no_evil: